Management/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+spring.mvc.view.prefix=/WEB-INF/view/
+spring.mvc.view.suffix=.jsp
+logging.level.org.org.springframework.web=DEBUG
```

Comment:
Why are you interested in this information?

Reply:
Deleted, just for development purpouse